release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -175,6 +175,52 @@ With this release, you can access the NMstate Operator and resources such as the
 [id="ocp-4-14-storage"]
 === Storage
 
+[id="ocp-4-14-storage-standardized-sts-config"]
+==== Standardized STS configuration workflow
+{product-title} 4.14 provides a streamlined and standardized procedure to configure Security Token Service (STS) with the AWS Elastic File Storage (EFS) Container Storage Interface (CSI) Driver Operator.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc#efs-sts_persistent-storage-csi-aws-efs[Configuring AWS EFS CSI Driver Operator with Security Token Service].
+
+[id="ocp-4-14-storage-rwod-access-mode"]
+==== Read Write Only Pod access mode (Technology Preview)
```

Comment:
Can we please update "Read Write Only Pod" to "Read Write Once Pod"

Reply:
+1 @lpettyjo

Reply:
Fixed!